zio-http/src/main/scala/zio/http/LeakDetectionLevel.scala

```diff
@@ -0,0 +1,23 @@
+package zio.http
+
+import io.netty.util.ResourceLeakDetector
+
+sealed trait LeakDetectionLevel {
```

Comment:
If "small" config data types like this appear _only_ in `ServerConfig`, it's acceptable (even desirable) to move them into the `ServerConfig` companion object, to avoid cluttering the top-level namespace with irrelevant classes.

Reply:
done